docs/applications/nublado/updating-recommended.rst

```diff
@@ -0,0 +1,44 @@
+##############################################
+Updating the recommended Notebook Aspect image
+##############################################
+
+The ``recommended`` tag for JupyterLab images is usually a recent weekly image.
+The image tagged ``recommended`` is guaranteed by SQuaRE to be compatible with other services and materials, such as tutorial or system testing notebooks, that we make available on RSP deployments.
+
+Because this process requires quite a bit of checking and sign-off from multiple stakeholders, it is possible that approving a new recommended version may take more than the two weeks (for most deployments) it takes for a weekly image to roll off the default list of images to pull.
+This can cause the RSP JupyterHub options form to display empty parentheses rather than the correct target version when a user requests a lab container.
+
+This document explains the process for moving the ``recommended`` tag, and how to circumvent that display bug by changing nublado's ``values-<instance>.yaml`` for the appropriate instance when moving the ``recommended`` tag.
```

Comment:
calling it a bug and circumventing it feels awkward in a deployment doc.  It might make more sense to explain what bug you're talking about.  For example:

and how to circumvent the issue where the recommended image in the jupyterhub lab creation UI shows an empty set of parenthesis next to recommended

or something like that?

Reply:
Or, better still, since it doesn't actually do that anymore, just cut the bits about the empty parens.